Engineering-Reference/M365DestinationAO.md

```diff
@@ -77,87 +77,85 @@ Default: Blank
 
 The access key for the Azure storage account.
 
-Change Conditions: Only avialable if the option SharePoint Migration API is enabled and SharePoint Storage is set to Azure Hosted. 
+Change Conditions: Only available if the option SharePoint Migration API is enabled and SharePoint Storage is set to Azure Hosted. 
 
 ### Retry Backoff (seconds) <a name="sretryback"></a>
 {: .no_toc }
 Default Value: 3
 
 SharePoint Migration API Retry Backoff in seconds.
 
-Change Conditions: ??
+Change Conditions: Retry backoff time can be changed. Can cause performance issues, not recommended.
 
 ### SharePoint Admin Url <a name="shareadurl"></a>
 {: .no_toc }
 Default Value: None
 
 The Url for the SharePoint admin center e.g. https://tenant-admin.sharepoint.com
 
-Change Conditions: ??
-
 ### Timeout <a name="stimeout"></a>
 {: .no_toc }
 Default Value: 1800000
 
 The timeout that will apply to communications with the SharePoint server.
 
-Change Conditions: ??
+Change Conditions: Timeout value (miliseconds) can be changed. Can cause performance issues, not recommended.
 
 ### Preserve File Created and Modified Dates <a name="persfilemoddate"></a>
 {: .no_toc }
 Default Value: On
 
 Preserve the Original File Created and Modified Dates.
 
-Change Conditions: ??
+Change Conditions: Disable to not preserve creation and modified dates of migrated files.
 
 ### Provision Sites <a name="provsites"></a>
 {: .no_toc }
 Default Value: On
 
 Provision any Sites that do not exist.
 
-Change Conditions: ??
+Change Conditions: Disable to to provision Sharepoint sites.
 
 ### Hybrid Environment <a name="hybridenv"></a>
 {: .no_toc }
 Default Value: On
 
 Allows a custom 'SharePoint Admin Url' and 'SharePoint My Sites Url' to be entered for hybrid migration.
 
-Change Conditions: ??
-
 ### SharePoint Storage <a name="sharstor"></a>
 {: .no_toc }
 Default Value: Office 365
 
-Choose whether the document should be shared as per the source file ACLs if they can be resolved to email addresses (see documentation).
+Microsoft’s Import Migration API requires the use of an Azure container for temporary storage. When (Office 365) is selected files and manifests will be uploaded to containers provided by Microsoft. Azure Hosted allows you to provide your own storage/containers.
+
+CloudM takes a snapshot of your destination using the Export Migration API which utilises storage containers. 'Storage Account Name' and 'Storage Account Key' are required for both (Office 365 and Azure Hosted).
 
-Change Conditions: ??
+Change Conditions: Choose the storage used, Office 365 or Azure Hosted.
```

Comment:
office does not have storage containers.   I think this is meant as choose Microsoft free storage account or provide your own azure storage account.

Reply:
done